spinalcordtoolbox/deepseg_/gm.py

```diff
@@ -1,10 +1,15 @@
-# This is the interface API for the deepseg_gm model
-# that implements the model for the Spinal Cord Gray Matter Segmentation.
-#
-# Reference paper:
-#     Perone, C. S., Calabrese, E., & Cohen-Adad, J. (2017).
-#     Spinal cord gray matter segmentation using deep dilated convolutions.
-#     URL: https://arxiv.org/abs/1710.01269
+"""
+API for the deepseg_gm model that implements the model for the
+Spinal Cord Gray Matter Segmentation
```

Comment:
Tiny nit, but could we standardize the strings of the three `deepseg/_module.py` docstrings?

```suggestion
API for functions used in the sct_deepseg_gm CLI script
```